tutormfe/templates/mfe/build/mfe/env.config.jsx

```diff
@@ -0,0 +1,48 @@
+{{- patch("mfe-env-config-static-imports") }}
+
+{{- patch("mfe-env-config-head") }}
+
+async function getConfig () {
+  let config = {};
+
+  try {
+    /* We can't assume FPF exists, as it's not declared as a dependency in all
+     * MFEs, so we import it dynamically. In addition, for dynamic imports to
+     * work with Webpack all of the code that actually uses the imported module
+     * needs to be inside the `try{}` block.
+     */
+    const { DIRECT_PLUGIN, PLUGIN_OPERATIONS } = await import('@openedx/frontend-plugin-framework');
+    {{- patch("mfe-env-config-dynamic-imports") }}
+
+    config = {
+      pluginSlots: {
+        {%- for _, slot_name, slot_config in iter_plugin_slots("all") %}
+        {{ slot_name }}: {
+          keepDefault: true,
+          plugins: [
+            {{- slot_config }}
+          ]
+        },
+        {%- endfor %}
+      }
+    };
+
+    {%- for app_name, app in iter_mfes() %}
+    if (process.env.npm_package_name == '@edx/frontend-app-{{ app_name }}') {
+      {{- patch("mfe-env-config-dynamic-imports-{}".format(app_name)) }}
+
+      {%- for _, slot_name, slot_config in iter_plugin_slots(app_name) %}
+      config.pluginSlots.{{ slot_name }}.plugins.push(
+        {{- slot_config }}
+      );
+      {%- endfor %}
+    }
+    {%- endfor %}
+
+    {{- patch("mfe-env-config-tail") }}
```

Comment:
nitpick: The "-tail" and "-head" suffixes are not used for other patch statements in Tutor. For instance, the patches in openedx/Dockerfile use "-pre", "-post", "-final". I'm not sure which convention is best, but we should probably follow the existing one.